tests/Remove-DbaLinkedServer.Tests.ps1

```diff
@@ -17,56 +17,85 @@ Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
-            $results = Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer "LS1_$random"
+            Remove-DbaLinkedServer -SqlInstance $script:instance2 -LinkedServer $linkedServerName1 -Confirm:$false
+            $results = Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer $linkedServerName1
             $results | Should -BeNullOrEmpty
         }
 
         It "Tries to remove a non-existent linked server" {
-            Remove-DbaLinkedServer -SqlInstance $script:instance2 -LinkedServer "LS1_$random" -Confirm:$false -WarningVariable warnings
-            $warnings | Should -BeLike "*Linked server LS1_$random does not exist on $($instance2.Name)"
+            Remove-DbaLinkedServer -SqlInstance $script:instance2 -LinkedServer $linkedServerName1 -Confirm:$false -WarningVariable warnings
+            $warnings | Should -BeLike "*Linked server $linkedServerName1 does not exist on $($instance2.Name)"
         }
 
         It "Removes a linked server using a server from a pipeline and a linked server from a pipeline" {
-            $results = Get-DbaLinkedServer -SqlInstance $script:instance2 -LinkedServer "LS2_$random"
+            $results = Get-DbaLinkedServer -SqlInstance $script:instance2 -LinkedServer $linkedServerName2
             $results.Length | Should -Be 1
-            Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer "LS2_$random" | Remove-DbaLinkedServer -WarningVariable warn -Confirm:$false
+            Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer $linkedServerName2 | Remove-DbaLinkedServer -WarningVariable warn -Confirm:$false
             $warn | Should -BeNullOrEmpty
-            $results = Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer "LS2_$random"
+            $results = Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer $linkedServerName2
             $results | Should -BeNullOrEmpty
 
-            $results = Get-DbaLinkedServer -SqlInstance $script:instance2 -LinkedServer "LS3_$random"
+            $results = Get-DbaLinkedServer -SqlInstance $script:instance2 -LinkedServer $linkedServerName3
             $results.Length | Should -Be 1
-            $instance2 | Remove-DbaLinkedServer -LinkedServer "LS3_$random" -Confirm:$false
-            $results = Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer "LS3_$random"
+            $instance2 | Remove-DbaLinkedServer -LinkedServer $linkedServerName3 -Confirm:$false
+            $results = Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer $linkedServerName3
+            $results | Should -BeNullOrEmpty
+        }
+
+        It "Removes a linked server that requires the -Force param" {
+            Get-DbaLinkedServer -SqlInstance $instance2 -LinkedServer $linkedServerName4 | Remove-DbaLinkedServer -Confirm:$false -WarningVariable warnings
+            $warnings | Should -BeLike "*There are still remote logins or linked logins for the server*"
```

Comment:
This should be removed, or put it a unique it block for testing the warning message. If this fails it will appear in the results that the use of `-Force` is broken.

Reply:
I moved this into its own test.